### PR TITLE
Add news post for playtest-20160403

### PR DESCRIPTION
**Pull request by obrakmann**
First draft. I'm not sure myself how happy I am with it, so have at it.

### CODE REVIEW

**Comment by pchote** [comment]
My initial impression is that this covers basically all the key points, so good job!

I'll spend some time tonight to come up with suggestions to compress and clarify some of the wording.

File: content/news/playtest-20160403.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+![Worm trails](/images/news/20160403-d2k-worm-trail.png)
+<p>
+Worms have received some polish and now leave a trail similar to the one from the original game.
+</p>
+</div>
+
+<hr />
+## Mapping and modding changes
+<hr />
+Maps have seen many changes since the release. So many changes, in fact, that the map format number was incremented four times &mdash; from 7 to 11 &mdash; during this development cycle!
```
Comment (pchote) [inline content/news/playtest-20160403.md]: IMO the underlying map format details aren't important enough to players or modders to justify ~15% of the news post text area.  I think we can reduce this down to a couple of sentences (paraphrased) "the format changed a bunch, which we'll talk about later, but for now you just need to know that the game won't automatically upgrade them – resource center has options for upgrading and downloading new versions".

Comment (pchote) [inline content/news/playtest-20160403.md]: @abcdefg30 has been trying to guilt me into writing a wiki page on the map format.  If we added that then we could link to it from here.

File: content/news/playtest-20160403.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+While there are the usual bug fixes and improvements, the biggest changes this time around are under the hood and will be felt foremost by server operators, modders and mappers. In fact, some of these changes are so fundamental that we will address them in-depth in the articles covering future playtests. To ensure that there will be enough dedicated servers available for playtesting, we will cover that topic below.
+
+Just to whet your appetite, dedicated servers no longer need to sync maps with the resource site, and it is now possible to add third-party mods as a single package (".oramod") to your installation.
+
+But one thing after the other...
```
Comment (pchote) [inline content/news/playtest-20160403.md]: There's a bit of duplication across these paragraphs, i'll see if I can come up with a compressed version.

**Comment by pchote** [comment]
Can we also include a mention of the website about page and games page improvements?

File: content/news/playtest-20160403.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+* Custom rules, weapon or sequence definitions can now be stored in external files.
+
+Support for third-party mods will significantly improve with the next release. It is now possible to create an archive of your whole mod (called ".oramod"). Installing it on other computers is then just a matter of placing that archive in the "mods" subdirectory of the Support folder.
+
+The news post for the second playtest will detail what changes you need to make to your mod to make it compatible with the upcoming release and to enable the .oramod packaging. All shipped mods have already been converted, so feel free to take a look at those.
+
+In the future, OpenRA will feature an ingame mod browser that will allow you to download and install mods from the resource site.
+
+<hr />
+## Dedicated server changes
```
Comment (pchote) [inline content/news/playtest-20160403.md]: :+1: for the wording in this section.  Maybe include a screenshot of the server output running in a terminal window (showing a couple of people joining, changing map, starting and ending a game) to break up the text between this and the next section?

**Comment by obrakmann** [comment]
Revised some passages, added screenshots to the other sections.

> Can we also include a mention of the website about page and games page improvements?

The text is already too long, and the about page isn't really of interest to old users. I'll see if I can fit the games page into some context.

File: content/news/playtest-20160403.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+![Worm trails](/images/news/20160403-d2k-worm-trail.png)
+<p>
+Worms have received some polish and now leave a trail similar to the one from the original game.
+</p>
+</div>
+
+<hr />
+## Mapping and modding changes
+<hr />
+Maps have seen many changes since the release. So many changes, in fact, that the map format number was incremented four times &ndash; from 7 to 11 &ndash; during this development cycle!
```
Comment (pchote) [inline content/news/playtest-20160403.md]: Can you encorporate some of the ideas from the following?

```
The way that the game handles maps has changed significantly since the last release.  We will elaborate on the changes and the reasons for them in a future post, but for now you only really need to know three things:

<ul markdown="1">
	<li>Maps from release-20151224 and earlier must be upgraded before they will work</li>
	<li>Maps are now installed into per-version subdirectories &mdash; like replays</li>
	<li>Installing maps in-game and using the map editor works as normal</li>
</ul>

We have automatically upgraded all the maps currently available on [the OpenRA Resource Center](http://resource.openra.net), and have added an option for you to manually upgrade your maps manually.
```

File: content/news/playtest-20160403.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+In the future, OpenRA will feature an ingame mod browser that will allow you to search for and install mods directly from the Resource Center.
+
+<hr />
+## Dedicated server changes
+<hr />
+Dedicated servers have seen quite a few improvements during this cycle, which make them altogether much easier to run.
+
+<div style="font-family: monospace; border-radius: 4px; background-color: #272d2c; padding:10px; margin:10px;">
+<em>
+$ ./OpenRA.Server.exe<br />
```
Comment (pchote) [inline content/news/playtest-20160403.md]: Could you add some demonstration parameters, like `Game.Mod=ra Server.Name="My Server" Server.AdvertiseOnline=True`? If you reduce the font size for the div to 85% it will fit on one line, and help distinguish the whole block from the rest of the text.  Or substitute other parameters as you want.

File: content/news/playtest-20160403.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+Support for third-party mods will significantly improve with the next release. It is now possible to create an archive of your whole mod (called ".oramod"). Installing it on other computers is then just a matter of placing that archive in the "mods" subdirectory of the Support folder.
+
+The news post for the second playtest will detail what changes your mod needs to make it compatible with the upcoming release and to enable the .oramod packaging. All shipped mods have already been converted, so feel free to take a look at those.
+
+In the future, OpenRA will feature an ingame mod browser that will allow you to search for and install mods directly from the Resource Center.
+
+<hr />
+## Dedicated server changes
+<hr />
+Dedicated servers have seen quite a few improvements during this cycle, which make them altogether much easier to run.
```
Comment (pchote) [inline content/news/playtest-20160403.md]: Can we make this a couple of words shorter so it doesn't overflow to the next line? There's also a bit of a running theme "Quite a few..." / "Maps have seen many changes..." / "Dedicated servers have seen quite a few..." that we could avoid.

File: content/news/playtest-20160403.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+[2016-04-02T16:37:38] Player has disconnected.<br />
+[2016-04-02T16:37:47] Antares has disconnected.<br />
+[2016-04-02T16:37:54] Newbie has disconnected.<br />
+No one is playing, shutting down...<br />
+Starting a new server instance...<br />
+[2016-04-02T16:37:55] Master server communication established.<br />
+</em>
+</div>
+
+First and foremost, the biggest change is that dedicated servers no longer need to sync maps with the Resource Center. Especially when you were running multiple servers of different versions, this could quickly become a headache. This will also mean that the distinction between synced and unsynced servers will be gone. That's right: all maps will be available on all servers, immediately after upload to the Resource Center, without any further action required.
```
Comment (pchote) [inline content/news/playtest-20160403.md]: This is more of a "but I would have done it that way" than a complaint, so feel free to ignore it, but this may be clearer as a bullet point list.

File: content/news/playtest-20160403.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+<hr />
+## Ingame changes
+<hr />
+
+Quite a few long-standing annoyances have been addressed during the last cycle. Just to name a few, players will no longer be kicked from the server after installing a map, walls now stop bullets reliably, and husks can be targeted on refineries.
+
+Other changes and new features include:
+
+* A couple of issues related to left-click controls have been solved.
```
Comment (pchote) [inline content/news/playtest-20160403.md]: My experience is that people stop paying attention half way down the list, so can we put the most exciting things first?
IMO the order should be:
```
* By popular demand, a tech level without super-powers has been added to the Red Alert mod.
* Zooming is now possible by holding CTRL while scrolling your mouse wheel. For spectators and in the map editor, two additional zoom steps are available.
* Canceling an entire build queue at once is now possible by using CTRL + right click.
* Static defenses will now stop tracking their targets once it is out of range. This also sometimes prevented camo pillboxes from cloaking.
* Line-building of walls can now be disabled by holding the Shift key while placing the wall element.
* A couple of issues related to left-click controls have been solved.
* Engineers now have a lower selection priority so that they will not be selected along with your regular combat units.
* Fragile diplomacy has been removed.
* Lots of minor fixes, UI polish and performance improvements.
```

File: content/news/playtest-20160403.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+* Static defenses will now stop tracking their targets once it is out of range. This also sometimes prevented camo pillboxes from cloaking.
+* Canceling an entire build queue at once is now possible by using CTRL + right click.
+* Line-building of walls can now be disabled by holding the Shift key while placing the wall element.
+* Zooming is now possible by holding CTRL while scrolling your mouse wheel. For spectators and in the map editor, two additional zoom steps are available.
+* Engineers now have a lower selection priority so that they will not be selected along with your regular combat units.
```
Comment (pchote) [inline content/news/playtest-20160403.md]: Would be good if we can reword this to fit on one line.

Maybe `* Engineers will no longer be selected when you drag-select a group of combat units.`?

File: content/news/playtest-20160403.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+
+Another change is that there is now a dedicated executable for dedicated servers, `OpenRA.Server.exe` (and a matching `/usr/bin/openra-server` on Linux). As a consequence, the old `Server.Dedicated` and `Server.DedicatedLoop` command-line arguments no longer exist. If you are using custom scripts to start your server instances, you will need to adapt them accordingly.
+
+One last, unrelated change is that the `Server.AllowBots` flag has been removed in favor of a `Server.DisableSinglePlayer` flag, which will allow people to play with bots on servers, but only if there is more than one human player in the game. `Server.DisableSinglePlayer` defaults to `false`, so be sure to set it if desired.
+
+<hr />
+So much for a general overview of the upcoming release. If you want to see the detailed list of changes, have a look at [the changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/<hash>).
```
Comment (pchote) [inline content/news/playtest-20160403.md]: It would be good if we could shorten this to a single line.

File: content/news/playtest-20160403.md [hunk 11]
```diff
@@ -0,0 +1,118 @@
+We are pleased to announce the first playtest leading up to our next stable release.
+
+Our development branch currently contains more than five months worth of changes, many of them very fundamental. As such, we have finally entered the feature freeze phase of the development cycle, where we stop adding features and focus on fixing bugs found during playtesting. This is where *you* come into play! We need *your* help to give this playtest as wide a test coverage as possible to ensure another successful stable release of OpenRA! We therefore ask you to please report any and all bugs and problems you encounter on [our bug tracker](http://bugs.openra.net), in the OpenRA [IRC channel](/community/), or here in the comments. We will strive to fix as many bugs as possible before a final release. Thank you for helping to improve OpenRA!
```
Comment (pchote) [inline content/news/playtest-20160403.md]: "many of them very fundamental" / "biggest changes this time around are under the hood" / "some of these changes are so fundamental" feels like you're stressing the point a bit *too* much :)

File: content/news/playtest-20160403.md (hunk 2 above)
Comment (pchote) [inline content/news/playtest-20160403.md]: My version would look something like
```
We are pleased to announce the first playtest for our next stable release!

We have finally entered the feature freeze phase of the development cycle, where we stop adding features and focus on fixing bugs found during playtesting. This is where *you* come into play! We need *your* help to give this playtest as wide a test coverage as possible to ensure another successful stable release of OpenRA!

We therefore ask you to please report any and all bugs and problems you encounter on [our bug tracker](http://bugs.openra.net), in the OpenRA [IRC channel](/community/), or here in the comments. We will strive to fix as many bugs as possible before a final release. Thank you for helping to improve OpenRA!

While there are the usual bug fixes and improvements, a big focus over the last six months has been reworking several fundamental parts of the OpenRA game engine, with changes that will be felt foremost by server operators, modders and mappers. In fact, some of these changes are so fundamental that we will address them in-depth in the articles covering future playtests.

But one thing after the other&hellip;
```
Feel free to take, modify, or ignore.

File: content/news/playtest-20160403.md [hunk 12]
```diff
@@ -0,0 +1,118 @@
...
+
+<div style="font-family: monospace; border-radius: 4px; background-color: #272d2c; padding:10px; margin:10px;">
+<em>
+$ ./OpenRA.Server.exe<br />
+Starting dedicated server for mod: ra<br />
+[2016-04-02T16:06:11] Master server communication established.<br />
+[2016-04-02T16:10:56] Player has joined the game.<br />
+[2016-04-02T16:11:49] Newbie has joined the game.<br />
+[2016-04-02T16:12:16] Antares changed the map to Doughnut.<br />
+Game started<br />
```
Comment (pchote) [inline content/news/playtest-20160403.md]: https://github.com/OpenRA/OpenRA/pull/11025

**Comment by obrakmann** [comment]
Updated.

File: content/news/playtest-20160403.md [hunk 13]
```diff
@@ -0,0 +1,118 @@
...
+* Zooming is now possible by holding CTRL while scrolling your mouse wheel. For spectators and in the map editor, two additional zoom steps are available.
+* Canceling an entire build queue at once is now possible by using CTRL + right click.
+* Static defenses will now stop tracking their targets once it is out of range. This also sometimes prevented camo pillboxes from cloaking.
+* Line-building of walls can now be disabled by holding the Shift key while placing the wall element.
+* A couple of issues related to left-click controls have been solved.
+* Engineers will no longer be selected when you drag-select a group of combat units.
+* Fragile diplomacy has been removed.
+* Lots of minor fixes, UI polish and performance improvements.
+
+As usual, both the Tiberian Dawn and Red Alert mod have received some balance changes. Additionally, a number of Red Alert maps have been revised to be more balanced. A complete list of all these changes can be found [in the changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/<hash>).
```
Comment (pchote) [inline content/news/playtest-20160403.md]: I guess we'll want this to link to a specific changelog revision.

**Comment by obrakmann** [comment]
Updated playtest id and permalink to the changelog.

**Comment by pchote** [comment]
:+1:
